test/metabase/api/action_test.clj

```diff
@@ -43,6 +43,8 @@
                                  :template {:method "GET"
                                             :url "https://example.com/{{x}}"}
                                  :parameters [{:id "x" :type "text"}]
+                                 :public_uuid (str (UUID/randomUUID))
+                                 :made_public_by_id 1
```

Comment:
This led me down an extensive rabbit hole since it turns out we just accept these params even though they're not mentioned in the schema (see discussion on Slack).

Anyway, I guess this test ensures that `public_uuid` and `made_public_by_id` are coming back in the `response`, so…fair play.